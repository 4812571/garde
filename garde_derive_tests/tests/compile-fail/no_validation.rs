#[derive(garde::Validate)]
struct Test<'a> {
    field: &'a str,
}

fn main() {}
