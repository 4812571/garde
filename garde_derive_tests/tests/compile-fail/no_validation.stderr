error: field has no validation, use `#[garde(skip)] if this is intentional
 --> tests/compile-fail/no_validation.rs:3:5
  |
3 |     field: &'a str,
  |     ^^^^^^^^^^^^^^
